test/common/router/route_fuzz_test.cc

```diff
@@ -36,13 +36,24 @@ cleanRouteConfig(envoy::api::v2::RouteConfiguration route_config) {
   auto internal_only_headers = clean_config.mutable_internal_only_headers();
   std::for_each(internal_only_headers->begin(), internal_only_headers->end(),
                 [](std::string& n) { n = Fuzz::replaceInvalidCharacters(n); });
-  // Remove invalid characters in each virtual host.
+  // Remove invalid characters in each virtual host and in their routes' cluster header names.
```

Comment:
Can you add a comment which line is exactly doing this?

Reply:
Done, thank you!